src/media/css/categories.styl

```diff
@@ -107,13 +130,24 @@ for $cat in $categories {
         }
     }
     .subheader.cat-header h1 {
-        margin: 0 0 0 20px;
-        padding: 48px 0;
+        $header-padding = 48px;
```

Comment:
ngoke prefers to have declarations at the top of the file. I like them closer to where they are used but we might want to be consistent and move these up.